examples/models/bert/distilbert.py

```diff
@@ -0,0 +1,31 @@
+from transformers import DistilBertForSequenceClassification, DistilBertModel
+
+class DistilBertClassifier(DistilBertForSequenceClassification):
+    def __init__(self, config):
+        super().__init__(config)
+        self.d_out = config.num_labels
```

Comment:
Here and in line 21, where is `self.d_out` used?

Reply:
I removed it.